_layouts/home.html

```diff
@@ -35,12 +35,20 @@
     </div>
   </div>
 </section>
+{% assign featuredprojects = (site.projects | where: "featured" , "homepage") %}
+{% if featuredprojects.size == 0 %}
```

Comment:
Logic to show and hide the featured section based on if there are any featured items to show or not